cmd/olm/cleanup.go

```diff
@@ -53,6 +56,10 @@ func cleanup(logger *logrus.Logger, c operatorclient.ClientInterface, crc versio
 	if err := cleanupOwnerReferences(c, crc); err != nil {
 		logger.WithError(err).Fatal("couldn't cleanup cross-namespace ownerreferences")
 	}
+
+	if err := checkAPIServiceLabels(c.ApiregistrationV1Interface()); err != nil {
```

Comment:
Rather than `checkAPIServiceLabels` could we name this `ensureAPIServiceLabels`? To me, `Check` suggests a that a `boolean` would be returned.